docs/_templates/layout.html

```diff
@@ -1,33 +1,39 @@
 {% extends '!layout.html' %}
 
+{% block extrahead %}
+<script>
+  (function (h, o, t, j, a, r) {
+    h.hj = h.hj || function () { (h.hj.q = h.hj.q || []).push(arguments) };
+    h._hjSettings = { hjid: 1819338, hjsv: 6 };
```

Comment:
can this id come from env? 

this section can probably go to footer as we don't want to block content rendering on this.

Reply:
I'll check out getting the id from env.

This can stay in head, hotjar docs says its async.

Reply:
it will still increase page load time because resources are limited on devices, especially mobile.

Reply:
Moved to footer 👍